modules/nf-core/bftools/showinf/meta.yml

```
@@ -0,0 +1,50 @@
+name: "bftools_showinf"
+description: "Extract OME xml data from OME-tif"
+keywords:
+  - metadata
+  - ome-tif
+  - ome-tiff
+  - imaging
+  - bioinformatics tools
+tools:
+  - "bftools":
+      description: "Suite of tools to handle several imaging protocols."
+      homepage: "https://www.openmicroscopy.org/bio-formats"
+      documentation: "https://bio-formats.readthedocs.io/en/stable/users/index.html"
+      tool_dev_url: "https://github.com/ome/bioformats"
```

```suggestion
      tool_dev_url: "https://github.com/ome/bioformats"
      doi: "10.5281/zenodo.7628291"
```

Perhaps you can add a DOI here? What do you think?
e.g. https://doi.org/10.5281/zenodo.7628291